src/precice/SolverInterface.cpp

```diff
@@ -331,6 +332,11 @@ MeshHandle SolverInterface:: getMeshHandle
   return _impl->getMeshHandle ( meshName );
 }
 
+const char * getVersionInformation()
```

Comment:
Is `const char*` mandated by xSDK? Otherwise, in a C++ interface, I expect a `std::string`.

Reply:
I agree. This will make it also easier to parse the string at the call-site.